HomeWork_6/FiveWords.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Class contains method to identify if line has five word
+ * and method that prints result.
+ *
+ * @author Nikita Zenchik
+ */
+public class FiveWords {
+    /**
+     * Method takes String line and finds if it has more then five words.
+     *
+     * @param inputline string line.
+     * @return True if line contains five or less words .
+     * @return False if line contains more then five words.
+     */
+    public static boolean howManyWords(String inputline) {
```

Comment:
Why static? Invalid name - the answer to question howManyWords is number, not true or false